src/index/IndexImpl.Text.cpp

```diff
@@ -803,5 +804,4 @@ void IndexImpl::storeTextScoringParamsInConfiguration(
     configurationJson_["b-and-k-parameter-for-text-scoring"] =
         std::make_pair(0.75, 1.75);
   }
-  writeConfiguration();
```

Comment:
I assume this must be the actual bug fix, right? It no longer overwrites the config when loading the CLI params. And now it doesn't.